presentation-compiler/src/main/dotty/tools/pc/PcDefinitionProvider.scala

```diff
@@ -123,9 +123,12 @@ class PcDefinitionProvider(
       case symbols @ (sym :: other) =>
         val isLocal = sym.source == pos.source
         if isLocal then
-          val defs =
-            Interactive.findDefinitions(List(sym), driver, false, false).filter(_.source == sym.source)
-          defs.headOption match
+          val (exportedDefs, otherDefs) =
+            Interactive.findDefinitions(List(sym), driver, false, false)
+              .filter(_.source == sym.source)
+              .partition(_.tree.symbol.is(Exported))
+
+          otherDefs.headOption.orElse(exportedDefs.headOption)  match
```

Comment:
What if instead of returning the head option we returned all the locations, in that `List(loc)` below, so that then downstream can show users a choice of location (i.e. the source of the export and the original)?

Other than that lgtm

Reply:
Generally, that makes sense. However, for this particular case it will cause an inconsistency in Metals. If the definition and user's cursor are not in the same file, pc will use Metals's symbol search, and that search will not be able to find the `export <smth>.*`.  So I am not sure that it would be better.